1.9/administration/logging/logging-api.md

```diff
@@ -1,46 +1,61 @@
 ---
-post_title: Logging API 
+post_title: Logging API
 feature_maturity: preview
 menu_order: 3
 ---
 
-# About the Logging API
+The Logging API exposes node, component, and container (task) logs.
 
-Use the Logging API to get the system logs from DC/OS.
+The Logging API is backed by the [DC/OS Log component](/docs/1.9/overview/architecture/components/#dcos-log), which runs on all nodes in the cluster.
 
-# Response format
+For more information about using the Logging API, see [Metrics](/docs/1.9/administration/metrics/).
```

Comment:
@karlkfi Are you intentionally pointing them to both Metrics and Logging for more information about using the Logging API? It seems to me that you should just point them to the Logging section and not to the Metrics, since they are two separate APIs.